be/src/olap/delete_handler.cpp

```diff
@@ -343,5 +343,17 @@ void DeleteHandler::finalize() {
     _is_inited = false;
 }
 
+void DeleteHandler::get_delete_conditions_after_version(int32_t version,
+        std::vector<const Conditions*>* delete_conditions) const {
+    if (delete_conditions == nullptr) {
```

Comment:
can this be nullptr?

Reply:
no, It will not. I will delete this check.